parse/parser_internal_test.go

```diff
@@ -2047,6 +2118,12 @@ func TestParseExpression(t *testing.T) {
 				},
 				0,
 			},
+			func() *symbol.Scope {
+				scope := symbol.NewScope()
+				scope.Set("a", &symbol.Integer{Name: &ast.Identifier{Value: "1"}})
```

Comment:
This does not impact on the tcs but..
the `"1"` in the `&ast.Identifier{Value: "1"}` **is not the value** of `a`, the `Value` should be name of the `a` so, should be `Value: "a"`

Reply:
except this LGTM! NICE WORK!!

Reply:
Ah, mistake :-)